### DISCUSSION TITLE
Yarn v4 changes question

### DISCUSSION THREAD

**Discussion by hugues-lacoursiere-zinnia** [yarnpkg/berry#5473] [Q&A] [answered]
Hey,
I was wondering, do we have a list of potential breaking changes from Yarn 3 to Yarn 4 so far, or some kind of WIP documentation?

I realize this is still a RC, but basically my team has been migrating our monorepo to Yarn PnP and we decided to go with v4 RC now, because v3 will often crash in vscode due to a ZipFS bug, which was fixed in v4 thanks to this https://github.com/yarnpkg/berry/pull/5178. 

I'd like to be aware if some behaviour or config has changed compared to the current v3 documentation, if that's possible.

Thanks

**Comment by arcanis** [comment, marked as answer]
This issue: https://github.com/yarnpkg/berry/issues/3591 and the changelog: https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md

**Reply by hugues-lacoursiere-zinnia** [reply]
Oh great didn't see that, thanks for the super quick answer!
